### DISCUSSION TITLE
fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-Dokodemo-Door-tunnel-as-relay-server/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4130] [Content Help کمک به اصلاح محتوا]
# fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-Dokodemo-Door-tunnel-as-relay-server/

The proxy manager platform.

https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-Dokodemo-Door-tunnel-as-relay-server/

**Comment by reza-naghizadeh** [comment]
سلام خسته نباشید من پنلم اوکی هستش و با ساب دامین کار میکنه و بعد فعال کردن
ipv6
لوکال تانل زدم بین سرور ایران و سرور اصلی ولی وقتی میخوام با ساب لینک سرور ایران وصل شم به پنل 
https
فعال نیست و بخاطر همین کانفینگ ها با تاتل کار نمیکنن
لظفا راهنمایی کنید چطوری رفع کنم این مشکل رو

**Reply by ReturnFI** [reply]
سلام.
برای گرفتن سرتیفیکیت پورت 80 تانل کنید

**Comment by Alireza-Aiden** [comment]
سلام خسته نباشید من برای tls port پورت 443و2053و8443 رو تعریف کردم و از cdn استفاده میکنم برای مستقیم الان میخوام از سرور ایران به عنوان میانی استفاده کنم با پورت 8443و2053 هردو تانل رو زدم و فعال هست وقتی اضافه کردم اصلا پینگ نمیده ممنون میشم راهنمایی کنید
